packages/wrangler/src/utils/print-bindings.ts

```diff
@@ -206,13 +206,22 @@ export function printBindings(
 					"allowed_destination_addresses" in emailBinding
 						? emailBinding.allowed_destination_addresses
 						: undefined;
+				const allowed_sender_addresses =
+					"allowed_sender_addresses" in emailBinding
+						? emailBinding.allowed_sender_addresses
+						: undefined;
+				let value =
+					destination_address ||
+					allowed_destination_addresses?.join(", ") ||
+					"unrestricted";
+
+				if (allowed_sender_addresses) {
+					value += ` - senders: ${allowed_sender_addresses.join(", ")}`;
```

Comment:
I don't think that adding a ` - senders:` makes for a great UI:
<img width="1012" height="711" alt="Screenshot 2025-09-16 at 15 18 32" src="https://github.com/user-attachments/assets/21af9a63-085e-4fb0-ae2e-d2b0c0ccfe78" />

possibly we should have some other ways of showing this, and maybe even truncating or hiding the information 🤔 

I can also see that testing for this is missing here (it was also missing before this PR anyways)

I propose that we just go with the current solution here (as I don't want to block/hinder the PR) and then I can improve the UI and add tests as a followup task (unless you'd like to @edevil)

What do you think? 🙂

Reply:
Yes, I agree that the TUI could probably be improved. I mainly added the sender addresses because we were already adding the other restrictions.
But lets leave it for a follow up, when we figure out how to improve it. We'll probably look at how other bindings are being displayed.